atomics/T1070.002/T1070.002.yaml

```diff
@@ -8,10 +8,28 @@ atomic_tests:
   supported_platforms:
   - macos
   - linux
+  input_arguments:
+    syslog_path:
+      description: path of syslog file to delete. On macos it's /var/log/system.log*, on linux, it's /var/log/syslog*. Also note for File events, that on macos, /var/ is a link to /private/var/.
+      type: String
+      default: /var/log/system.log
+    macos_audit_path:
+      description: path of audit file to delete
+      type: String
```

Comment:
Thanks for your contribution. Can you change this from `type: String` to `type: string` on line 14 and 18 to match the Schema ?

Reply:
np. done